.github/workflows/all-static-checks.yaml

```diff
@@ -88,6 +88,48 @@ jobs:
         run: |
           pip install pyyaml
           python tests/sweep_framework/framework/sweeps_workflow_verification.py
+  cmake-version:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
+    - name: Check for changed CMake files
+      id: changed-cmake-files
+      uses: tj-actions/changed-files@v45
+      with:
+        files: |
+          **/*.cmake
+          **/CMakeLists.txt
+    - uses: lukka/get-cmake@v3.30.5
```

Comment:
Would recommend using a SHA to directly reference the tag rather than the tag itself - just some small security hygiene

Reply:
Good point; this is a 3rd party reference.